src/app/core/models/consultant.ts

```diff
@@ -18,8 +18,7 @@ export interface Consultant {
     email: string;
     username: string;
     address: Address;
-    phone1: number;
-    phone2?: number;
+    phone: number;
```

Comment:
I think I just wrote something in the export-profile that used phone1. If we are going to change the model that will need to be updated.